lib/redcord.rb

```diff
@@ -1,9 +1,36 @@
+# frozen_string_literal: true
+
 # typed: strict
-module Redcord
-end
 
 require 'sorbet-runtime'
 
+module Redcord
+  extend T::Sig
+
+  @@configuration_blks = T.let(
+    [],
+    T::Array[T.proc.params(arg0: T.untyped).void],
```

Comment:
what if the config blocks have different lengths of arguments?

Reply:
The blocks are only meant to be called by us here @ line 27